vpr/src/route/rr_graph.cpp

```diff
@@ -271,7 +271,8 @@ static void connect_src_sink_to_pins(RRGraphBuilder& rr_graph_builder,
                                      const int j,
                                      t_rr_edge_info_set& rr_edges_to_create,
                                      const int delayless_switch,
-                                     t_physical_tile_type_ptr physical_type_ptr);
+                                     t_physical_tile_type_ptr physical_type_ptr,
+                                     bool is_remapped);
```

Comment:
is_remapped doesn't seem like a good name.  Suggest switches_remapped.
Add a Doxygen comment above this function; at least describe switches_remapped but better to describe it all.